public/app/core/utils/kbn.ts

```diff
@@ -584,8 +584,8 @@ kbn.valueFormats.flowcms = kbn.formatBuilders.fixedUnit('cms');
 kbn.valueFormats.flowcfs = kbn.formatBuilders.fixedUnit('cfs');
 kbn.valueFormats.flowcfm = kbn.formatBuilders.fixedUnit('cfm');
 kbn.valueFormats.litreh = kbn.formatBuilders.fixedUnit('l/h');
-kbn.valueFormats.flowlpm = kbn.formatBuilders.decimalSIPrefix('L');
-kbn.valueFormats.flowmlpm = kbn.formatBuilders.decimalSIPrefix('L', -1);
+kbn.valueFormats.flowlpm = kbn.formatBuilders.decimalSIPrefix('L/min');
+kbn.valueFormats.flowmlpm = kbn.formatBuilders.decimalSIPrefix('L/min', -1);
```

Comment:
Shouldn't this be `mL/min`?
```suggestion
kbn.valueFormats.flowmlpm = kbn.formatBuilders.decimalSIPrefix('mL/min', -1);
```

Reply:
You are correct, my mistake. 
I donät know if it should be ml/min or mL/min?

Reply:
I'm not sure either. Looking at the submenu title it's `mL/min`.